modules/apps/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/display/context/SelectSiteInitializerDisplayContext.java

```diff
@@ -150,7 +150,7 @@ private List<SiteInitializerItem> _getSiteInitializerItems()
 						themeDisplay.getCompanyId(), Boolean.TRUE, null),
 					layoutSetPrototype -> new SiteInitializerItem(
 						layoutSetPrototype, themeDisplay.getLocale())),
-				new SiteInitializerNameComparator(true));
+				SiteInitializerNameComparator.getInstance(true));
```

Comment:
@Valmir-unicap All the usages of this are passing `true`. Maybe we can remove the `_DESCENDING` instance?

@tinatian what do you think?

By the way, this is an `OrderByComparator` but it's only used in pure list sorting. Do we need to take any action about it?

cc @shuyangzhou

Reply:
if it is not used for db sorting, just change it to a plain Comparator